tasks/users.yml

```diff
@@ -13,7 +13,7 @@
     port: "{{postgresql_port}}"
     state: "{{ item.state | default('present') }}"
     login_user: "{{postgresql_admin_user}}"
-  no_log: true
+#  no_log: true
```

Comment:
```suggestion
  no_log: "{{postgresql_nolog}}"
```
❓

Reply:
Will fix that. It is a relic from some debugging.